pkg/loop/internal/relayer/pluginprovider/ext/ccip/offramp_test.go

```diff
@@ -57,8 +57,8 @@ func Test_OnchainConfig(t *testing.T) {
 		},
 	}
 
-	for _, tt := range tests {
-		tt := tt
+	for _, tt := range tests { // nolint
+		tt := tt // nolint
```

Comment:
what's going on here?

Reply:
lint doesn't like that the proto is copied because protos have a lock.